### PR TITLE
Always show current and next session of the day

### PR DESCRIPTION
**Pull request by holistic-developer**
This will show the current and upcoming session (given there are some on the current day) using the current time.
So on the 8th at 9:30 it would look like this 
image

On the 9th at 9:30 like this
image

And on smaller screens like this
image

### CODE REVIEW

File: src/components/pages/live/live.jsx [hunk 1]
```diff
@@ -3,14 +3,15 @@ import React from 'react';
 import { scheduleUrl } from '../../../constants/sessionize-app';
 import Button from '../../shared/button';
 
+import CurrentSessions from './current-sessions/current-sessions';
+
 const Live = () => (
-    <section className="safe-paddings container-md text-center grid grid-cols-2 sm:grid-cols-1 gap-4">
+    <section className="safe-paddings container-md text-center grid grid-cols-2 sm:grid-cols-1 gap-4 mb-4">
       <div className="col-span-2 sm:col-span-1 bg-gray-12 rounded-md p-2 flex-col inline-block justify-center">
         <h2>Schedule</h2>
```
Comment (KatharinaSick) [inline src/components/pages/live/live.jsx:11]: Up to you if you want to implement it, but should we rename this to Current/Next Sessions or something like that?

**Comment by KatharinaSick** [comment]
That's super cool! Thank you :)

File: src/components/pages/live/live.jsx (hunk 1 above)
Comment (holistic-developer) [reply src/components/pages/live/live.jsx:11]: How about just "Sessions" instead of Schedule?

**Comment by holistic-developer** [comment]
One more update.
This is how it will look at the end of the day (2024-10-08)
image

And here is how it will look when nothing is scheduled at the current day
image

I decreased the header size so it won't wrap on mobile

**Review by jgrumboe — APPROVED** [review]

**Pull request merged**